src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -115,5 +115,21 @@ describe("cloudscheduler", () => {
 
       expect(nock.isDone()).to.be.true;
     });
+
+    it("should error and exit if cloud scheduler create request fail", async () => {
+      nock(api.cloudschedulerOrigin)
+        .get(`/${VERSION}/${TEST_JOB.name}`)
+        .reply(404, { context: { response: { statusCode: 404 } } });
+      nock(api.cloudschedulerOrigin)
+        .post(`/${VERSION}/projects/test-project/locations/us-east1/jobs`)
+        .reply(400, { context: { response: { statusCode: 400 } } });
+
+      await expect(cloudscheduler.createOrReplaceJob(TEST_JOB)).to.be.rejectedWith(
+        FirebaseError,
+        ""
```

Comment:
Why is this testing that the error message contains an empty string? Can this be more specific?

Reply:
Done!